tasks/build/download_node_builds.js

```diff
@@ -68,17 +68,75 @@ module.exports = function (grunt) {
     var bytes = parseInt(resp.headers['content-length'], 10) || 'unknown number of';
     var mb = ((bytes / 1024) / 1024).toFixed(2);
     grunt.log.ok(`downloading ${platform.name} - ${mb} mb`);
+
+    await platform.downloadPromise;
```

Comment:
Now that we don't have to do bookeeping of active downloads between start and end tasks, we don't really need to store these promises in variables, we can simply `await` on the execution itself.